cluster/manifests/kube-metrics-adapter/deployment.yaml

```diff
@@ -42,6 +42,7 @@ spec:
         - --aws-external-metrics
         - --aws-region=eu-central-1
         - --aws-region=eu-west-1
+        - --aws-region=eu-north-1
```

Comment:
At some point we enabled support for scaling based on SQS queues in `eu-west-1` even if the cluster was in `eu-central-1`. That is why the config looks like this. We should double check if this is used at all and only enable the local region. I would propose that for now we change `--aws-region=eu-central-1` to `--aws-region={{.Cluster.Region}}` and drop the flag for `--aws-region=eu-north-1` such that we only have local region + `eu-west-1` which may go away again.

Reply:
Sure, lets do that for now. Updated.